{"name": "Авторизация с неверным faker.email() пользователя ", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range", "trace": "self = <test_login.TestLoginUser object at 0x1051b0550>, register_new_user_return_login_and_password = [], email = 'faker.email()'\n\n    @allure.title('Авторизация с неверным {email} пользователя ')\n    @allure.description('Запрос проверяет авторизацию с несуществующей почтой пользователя, но с корректным паролем')\n    @pytest.mark.parametrize('email', ['faker.email()', ' '], ids=['incorrect_email', 'empty_email'])\n    def test_login_user_incorrect_email_and_correct_password(self, register_new_user_return_login_and_password,\n                                                             email):\n        faker = Faker()\n        data = register_new_user_return_login_and_password\n        payload = {\n            \"email\": email,\n>           \"password\": data[1]\n        }\nE       IndexError: list index out of range\n\ntests/test_login.py:39: IndexError"}, "description": "Запрос проверяет авторизацию с несуществующей почтой пользователя, но с корректным паролем", "parameters": [{"name": "email", "value": "'faker.email()'"}], "start": 1723915771919, "stop": 1723915771955, "uuid": "656155d5-686e-4a35-9dfb-e915a608a53e", "historyId": "bd3432931a700b4d8d15ff4e79de9faa", "testCaseId": "7092bc473ffb96023fc9047acacdda00", "fullName": "tests.test_login.TestLoginUser#test_login_user_incorrect_email_and_correct_password", "labels": [{"name": "suite", "value": "Авторизация пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "Air-Anna"}, {"name": "thread", "value": "73159-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}